<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="GiaoDienChinh">
      <item>
       <layout class="QHBoxLayout" name="NutDieuKien">
        <item>
         <widget class="QPushButton" name="btnMenu">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resource.qrc">
            <normaloff>:/:/icons/store.png</normaloff>:/:/icons/store.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOrder">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resource.qrc">
            <normaloff>:/:/icons/bag.png</normaloff>:/:/icons/bag.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="NutTimKiem">
        <item>
         <widget class="QPushButton" name="ToanBo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Toàn Bộ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DoAn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Đồ Ăn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ThucUong">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Thức Uống</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DoGiaDung">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Đồ Gia Dụng</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="TimKiemSanPham">
        <item>
         <widget class="QLineEdit" name="SearchText">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Nhập tên sản phẩm cần tìm…</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BtnSearch">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Tìm</string>
          </property>
          <property name="icon">
           <iconset resource="Resource.qrc">
            <normaloff>:/:/icons/loupe.png</normaloff>:/:/icons/loupe.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockMenu">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>900</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::NoDockWidgetFeatures</set>
   </property>
   <property name="dockLocation">
    <enum>Qt::DockWidgetArea::LeftDockWidgetArea</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="Menu">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="ThongKe">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Thống Kê Lợi Nhuận</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/:/icons/report.png</normaloff>:/:/icons/report.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ThemHang">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Thêm Hàng Vào Kho</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/:/icons/add-product.png</normaloff>:/:/icons/add-product.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="KhachHang">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Khách Hàng</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/:/icons/rating.png</normaloff>:/:/icons/rating.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockOrder">
   <property name="minimumSize">
    <size>
     <width>668</width>
     <height>900</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="HoaDon">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidgeOrder">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="ChuanBiHoaDon">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="MainOrder">
           <item>
            <layout class="QVBoxLayout" name="Customer">
             <item>
              <widget class="QLineEdit" name="txtSearchCustomer">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="ifCustomer">
               <item>
                <widget class="QLabel" name="lblTenKhach">
                 <property name="text">
                  <string>Khách Lẻ</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lblDiemKhach">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="ifOrder">
             <item>
              <widget class="QTableView" name="tableViewOrder">
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>50</number>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>50</number>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>50</number>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TotalBefore">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TotalAfter">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="btnOrder_2">
             <item>
              <widget class="QPushButton" name="btnDungDiem">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Dùng 100.000 điểm (Giảm 2%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ExportOrder">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Xuất Hóa Đơn</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ThanhToan">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableView" name="tableLastBill"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="information">
           <item>
            <widget class="QLabel" name="TotalBefore_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="TotalAfter_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="method">
           <item>
            <widget class="QPushButton" name="btnCard">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Thanh Toán Bằng Thẻ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCash">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Thanh Toán Bằng Tiền Mặt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="btnBackOrDone">
           <item>
            <widget class="QPushButton" name="btnBack">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Quay Lại</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnExport">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Thanh Toán</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
